src/Sarif.Sarifer/TextBufferViewTracker.cs

```diff
@@ -60,14 +66,28 @@ public void RemoveTextView(ITextView textView)
             trackingInformation.Remove(textView);
             if (trackingInformation.Views.Count == 0)
             {
-                this.bufferToViewsDictionary.Remove(textView.TextBuffer);
+                this.bufferToViewsDictionary.TryRemove(textView.TextBuffer, out TextBufferViewTrackingInformation value);
                 trackingInformation.CancellationTokenSource.Dispose();
                 LastViewRemoved?.Invoke(
                     this,
                     new LastViewRemovedEventArgs(trackingInformation.Path));
             }
         }
 
+        /// <inheritdoc/>
+        public void UpdateTextView(ITextView textView, string path, string text)
```

Comment:
>string path [](start = 55, length = 11)

should we remove this since we are not using?

Reply:
done

---
In reply to: [564965225](https://github.com/microsoft/sarif-visualstudio-extension/pull/349#discussion_r564965225) [](ancestors = 564965225)